draft-ietf-quic-transport.md

```diff
@@ -549,70 +548,70 @@ See {{version-negotiation}} for a description of the version negotiation
+({{packet-handshake}}) packets.
 
-All cleartext packets contain the current QUIC version in the version field.
+All these packets use the long header and contain the current QUIC version in
+the version field.
 
-In order to prevent tampering by version-unaware middleboxes, Cleartext
-packets are protected with a connection and version specific key, as
-described in {{QUIC-TLS}}. This protection does not provide confidentiality
-or integrity against on-path attackers, but provides some level of
-protection against off-path attackers.
+In order to prevent tampering by version-unaware middleboxes, handshake packets
+are protected with a connection and version specific key, as described in
+{{QUIC-TLS}}. This protection does not provide confidentiality or integrity
+against on-path attackers, but provides some level of protection against
+off-path attackers.
 
 
-### Client Initial Packet {#packet-client-initial}
+### Initial Packet {#packet-initial}
 
-The Client Initial packet uses long headers with a type value of 0x02.
-It carries the first cryptographic handshake message sent by the client.
+The Initial packet uses long headers with a type value of 0x02.  It carries the
+first cryptographic handshake message sent by the client.
 
 The client populates the connection ID field with randomly selected values,
 unless it has received a packet from the server.  If the client has received a
 packet from the server, the connection ID field uses the value provided by the
 server.
 
-The packet number used for Client Initial packets is initialized with a random
-value each time the new contents are created for the packet.  Retransmissions of
-the packet contents increment the packet number by one, see
-({{packet-numbers}}).
+The packet number used for Initial packets is initialized with a random value
+each time the new contents are created for the packet.  Retransmissions of the
+packet contents increment the packet number by one, see ({{packet-numbers}}).
 
-The payload of a Client Initial packet consists of a STREAM frame (or frames)
+The payload of a Initial packet consists of a STREAM frame (or frames)
 for stream 0 containing a cryptographic handshake message, with enough PADDING
 frames that the packet is at least 1200 octets (see {{packetization}}).  The
 stream in this packet always starts at an offset of 0 (see {{stateless-retry}})
 and the complete cryptographic handshake message MUST fit in a single packet
 (see {{handshake}}).
 
-The client uses the Client Initial Packet type for any packet that contains an
-initial cryptographic handshake message.  This includes all cases where a new
-packet containing the initial cryptographic message needs to be created, this
-includes the packets sent after receiving a Version Negotiation
-({{packet-version}}) or Server Stateless Retry packet
-({{packet-server-stateless}}).
+The client uses the Initial Packet type for any packet that contains an initial
```

Comment:
Some capitalization inconsistency between "Initial Packet" and "Initial packet".  Might be worth a quick pass to decide which system you intend to use.